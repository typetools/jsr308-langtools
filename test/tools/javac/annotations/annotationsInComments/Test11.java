/*
 * @test
 * @bug 1234567
 * @summary Ignore split comments; test for Checker Framework Issue 239
 * @author Werner Dietl
 * @compile/fail/ref=Test11.out -XDrawDiagnostics Test11.java
 */

// Unrecognized comment
/*@*//*Annotation(Test11a.class)*/
class Test11a {}

// Recognized comment, unknown annotation
/*@Annotation(Test11.class)*/
class Test11b {}
