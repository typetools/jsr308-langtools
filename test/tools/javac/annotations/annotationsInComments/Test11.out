Test11.java:14:4: compiler.err.cant.resolve: kindname.class, Annotation, , 
1 error